.github/actions/build-dotnet/action.yml

```diff
@@ -84,10 +92,15 @@ runs:
       shell: pwsh
       working-directory: ${{ inputs.directory }}
       run: |
+        $buildSwitches = @'
+        ${{ inputs.dotnet-build-switches }}
+        '@
```

Comment:
```suggestion
            ${{ inputs.dotnet-build-switches }}
            '@
```

Reply:
I have indented `${{ inputs.dotnet-build-switches }}`, becuase every of its lines get trimmed anyway.
However you mustn't indent the `'@` (closing part of the here-string). It must be at the very beginning of the line otherwise it won't work. As far as I understand in YAML that means the same level of indentation as the first line of the snippet. (significant whitespace, spooky!) So that must stay.